Handling Events with Stencil

```diff
@@ -0,0 +1,17 @@
+import  { Component, Prop, State, h } from  '@stencil/core';
+
+@Component({
+   tag: 'my-rating',
+   styleUrl: 'my-rating-component.css',
```

Comment:
where is the css file? @Pawan85

Reply:
<!DOCTYPE html>
<html dir="ltr" lang="en">
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0">
  <title>Stencil Component Starter</title>
  <script type="module" src="/build/rating-stencil-component.esm.js"></script>
  <script nomodule src="/build/rating-stencil-component.js"></script>
  </head>
  <body>
    <my-rating id="myRatingComponent" max-value="6" value="2"></my-rating>
  </body>
  </html>